client/components/forms/sortable-list/index.scss

```diff
@@ -18,6 +18,7 @@
 
 	&.is-active > * {
 		box-shadow: 0 0 0 2px white, 0 0 0 4px $blue-medium;
+		z-index: 10;
```

Comment:
We don't normally use numbers for `z-index` but [this function](https://github.com/Automattic/wp-calypso/blob/a70587157b33783ba4d5e0e4a1e136680d459d51/assets/stylesheets/shared/functions/_z-index.scss). See jsdoc there for instructions. (Or in other words, I have no idea how to use that 😆)

Reply:
Oh, we're actually referencing `SortableList` in that file:

https://github.com/Automattic/wp-calypso/blob/0ec53e65acb3536ecec455a1f7dd0a66bed56448/assets/stylesheets/shared/functions/_z-index.scss#L143

Reply:
So the issue was that `.sortable-list__item.is-draggable.is-active` selector was too specific and only worked on the desktop version, where dragging is enabled.  
Cutting it down to `.sortable-list__item.is-active` ensures the active item is always the one in front.